### DISCUSSION TITLE
Is there a way to separate admin and normal signin emails stored in Firefox

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#2562] [Q&A]
This isn't exactly a Pocketbase issue, but maybe Pocketbase could do something to help.  I'm currently finding that Firefox wants to only save one email / password combination for both my `/_` admin panel and the normal custom stuff that I made.  Is there a way to signal to browsers that it's another separate section that should store a separate password combo?  Maybe an input or form attribute?  Maybe someone could chime in if something exists that I could implement myself, and/or if Pocketbase could also implement.

**Comment by ganigeorgiev** [comment]
I haven't tested it but it should be possible if you specify a different name attribute and/or set and explicit autocomplete value.

I'll consider for the next release to try to "isolate" the admin login form autocompletion.
